Mathlib/Topology/Order/Bounded.lean

```diff
@@ -1,40 +0,0 @@
-/-
-Copyright (c) 2023 Kalle Kytölä. All rights reserved.
-Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Kalle Kytölä
--/
-import Mathlib.Topology.Bornology.Basic
-import Mathlib.Topology.Instances.Real
-import Mathlib.Order.LiminfLimsup
-
-/-!
-# Relating order and metric boundedness
-
-In spaces equipped with both an order and a metric, there are separate notions of boundedness
-associated with each of the two structures. In specific cases such as ℝ, there are results which
-relate the two notions.
-
-## Tags
-
-bounded, bornology, order, metric
--/
-
-open Set Filter
-
-section Real
-
-lemma Filter.isBounded_le_map_of_bounded_range {ι : Type*} (F : Filter ι) {f : ι → ℝ}
```

Comment:
Why do you delete these lemmas instead of generalizing/deprecating them? I see that they should use `Filter.IsBoundedUnder`, what are other reasons?

Reply:
They are a simple combination of easier lemmas belonging to different areas, namely `Bornology.IsBounded.bddBelow`/`Bornology.IsBounded.bddAbove` and `isBoundedUnder_of` and some quantifier reshuffle. I did not deprecate them because
* I want to be able to remove the file directly
* I cannot state them using `ℝ` in `Topology.Order.Bornology`
* I don't want to have to think about how I should restate them
* I can't redirect to a single lemma within the deprecation
* They have been added for exactly one use anyway

Writing that message, I have now spent too much time thinking about those two lemmas.

Reply:
I think our current policy would lean towards deprecating them

Reply:
I have restored `Topology.Order.Bounded`